bedrock/mozorg/templates/mozorg/internet-health/index.html

```diff
@@ -45,6 +45,35 @@ <h1><span>{{ _('Internet Health') }}</span></h1>
     </div>
   </header>
 
+{% if l10n_has_tag('health_hub_data_detox_kit') %}
+  <section class="data-detox-kit">
+    <div class="content">
+      <div class="card-image-large">
+        <h2 class="section-title"><span>{{ _('Data Detox Kit') }}</span></h2>
+
+        <div class="card-content-wrapper">
+          <div class="card-content">
+            <a href="https://blog.mozilla.org/internetcitizen/2017/11/08/data-detox/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=ih-content-hub" data-link-name="Get your digital cleanse" data-link-type="link" data-link-position="data-detox-kit">
+              <h3 class="card-heading">{{ _('Get your digital cleanse') }}</h3>
```

Comment:
I wonder if using "cleanse" as a noun here will translate. Maybe this needs an l10n note to clarify? Maybe an alternative headline for localizers?